enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/IdentityVerificationStatusService.java

```diff
@@ -168,19 +172,18 @@ public IdentityVerificationStatusResponse checkIdentityVerificationStatus(Identi
                 } catch (PresenceCheckException e) {
                     logger.error("Checking presence verification failed, {}", ownerId, e);
                     idVerification.setErrorDetail(e.getMessage());
-                    idVerification.setStatus(IdentityVerificationStatus.FAILED);
+                    idVerification.setStatus(FAILED);
                     idVerification.setTimestampLastUpdated(ownerId.getTimestamp());
                 }
 
                 if (presenceCheckResult != null) {
                     evaluatePresenceCheckResult(ownerId, idVerification, presenceCheckResult);
                 }
             }
-        } else if (IdentityVerificationPhase.PRESENCE_CHECK.equals(idVerification.getPhase())
-                && IdentityVerificationStatus.VERIFICATION_PENDING.equals(idVerification.getStatus())) {
+        } else if (isPresenceCheckFinished(idVerification.getPhase(), idVerification.getStatus())) {
```

Comment:
The main change in _status service_, now FAILED and REJECTED are accepted to pass to the next step because of SCA.